examples/simulation/plot_simulated_raw_data_using_subject_anatomy.py

```diff
@@ -0,0 +1,181 @@
+    'auditory/right':
+        [('G_temp_sup-G_T_transv-lh', 200),
+         ('G_temp_sup-G_T_transv-rh', 100)],
+    'visual/left':
+        [('S_calcarine-lh', 100),
+         ('S_calcarine-rh', 200)],
+    'visual/right':
+        [('S_calcarine-lh', 200),
+         ('S_calcarine-rh', 100)],
+}
+
+annot = 'aparc.a2009s'
+
+# Load the 4 necessary label names.
+label_names = sorted(set(activation[0]
+                         for activation_list in activations.values()
+                         for activation in activation_list))
+region_names = list(activations.keys())
+
+#  Define the time course of the activity for each region to activate. We use a
+#  sine wave and it will be the same for all 4 regions.
+source_time_series = np.sin(np.linspace(0, 4 * np.pi, 100)) * 10e-9
+
+###############################################################################
+# Create simulated source activity
+# --------------------------------
+#
+# Here, SourceSimulator is used, which allows to specify where (label), what
+# (source_time_series), and when (events) event type will occur.
+#
+# We will add data for 4 areas, each of which contains 2 labels. Since add_data
+# method accepts 1 label per call, it will be called 2 times per area.
+# All activations will contain the same waveform, but the amplitude will be 2
+# times higher in the contralateral label, as explained before.
+#
+# When the activity occurs is defined using events. In this case, they are taken
+# from the original raw data. The first column is the sample of the event, the
+# second is not used. The third one is the event id, which is different for each
+# of the 4 areas.
+
+source_simulator = mne.simulation.SourceSimulator(src, tstep=tstep)
+for region_id, region_name in enumerate(region_names, 1):
+    events_tmp = events[np.where(events[:, 2] == region_id)[0], :]
+    for i in range(2):
+        label_name = activations[region_name][i][0]
+        label_tmp = mne.read_labels_from_annot(subject, annot,
+                                               subjects_dir=subjects_dir,
+                                               regexp=label_name,verbose=False)
+        label_tmp = label_tmp[0]
+        amplitude_tmp = activations[region_name][i][1]
+        source_simulator.add_data(label_tmp, amplitude_tmp*source_time_series,
+                                  events_tmp)
+
+# To obtain a SourceEstimate object, we need to use `get_stc()` method of
+# SourceSimulator class.
+stc_data = source_simulator.get_stc()
+
+###############################################################################
+# Simulate raw data
+#------------------
+#
+# Project the source time series to sensor space. Three types of noise will be
+# added to the simulated raw data:
+#     - multivariate Gaussian noise obtained from the noise covariance from
+#       the sample data
+#     - blink (EOG) noise
+#     - ECG noise
+# The source simulator can be given directly to the `simulate_raw` function.
```

Comment:
single backticks just make italics.  code should have 
```
``double_backticks``
```
although in this case it would be better to link to the function like this:
```
:func:`~mne.simulation.simulate_raw`
```
the tilde suppresses `mne.simulation.` so it renders the link as just `simulate_raw()` (it adds the parentheses automatically)

Reply:
```suggestion
# The :class:`~mne.simulation.SourceSimulator` can be given directly to the :func:`~mne.simulation.simulate_raw` function.
```